.vib/grafana/cypress/cypress/integration/grafana_test_suite.js

```diff
@@ -18,52 +15,54 @@ it('allows creating a dashboard with a panel', () => {
 
   cy.login();
   cy.visit('dashboard/new');
-  cy.contains('button', 'Add a new panel')
-    .should('be.visible').click();
+  cy.contains('button', 'Add a new panel').click();
   cy.get('[value*="Panel Title"]').clear().type(`${PANEL_TITLE}.${random}`);
-  cy.get('textarea#description-text-area')
-    .should('exist').type(`${PANEL_DESCRIPTION}.${random}`);
+  cy.get('textarea#description-text-area').type(
+    `${PANEL_DESCRIPTION}.${random}`
+  );
 
-  cy.contains('button','Apply')
-    .should('be.visible').click();
+  cy.contains('button', 'Apply').click();
 
-  cy.contains('.panel-title', PANEL_TITLE).should('be.visible');
+  cy.contains('.panel-title', PANEL_TITLE);
   cy.get('[aria-label*="Save dashboard"]').click();
-  cy.get('button[aria-label*="Close dialogue"]')
-    .should('be.visible');
+  cy.get('button[aria-label*="Close dialogue"]');
   cy.get('[name*="title"]').clear().type(`${DASHBOARD_TITLE}.${random}`);
-  cy.get('[type*="submit"]')
-    .should('be.visible').click();
+  cy.get('[type*="submit"]').click();
   verifySuccesOfAction();
-})
+});
 
 it('checks if it is possible to upload a dashboard as JSON file', () => {
   const DASHBOARD_TITLE = 'New Test dashboard';
 
   cy.login();
   cy.visit('dashboard/import');
-  cy.contains('label','Upload JSON').click();
-  cy.get('input[type=file]').selectFile('cypress/fixtures/test-dashboard.json',
-  {force:true});
-  cy.get('[data-testid*="data-testid-import-dashboard-title"]').clear().type(`${DASHBOARD_TITLE}.${random}`);
+  cy.contains('label', 'Upload JSON').click();
+  cy.get('input[type=file]').selectFile(
+    'cypress/fixtures/test-dashboard.json',
+    { force: true }
+  );
+  cy.get('[data-testid*="data-testid-import-dashboard-title"]')
+    .clear()
+    .type(`${DASHBOARD_TITLE}.${random}`);
   cy.get('[data-testid*="data-testid-import-dashboard-submit"]').click();
-  cy.visit("dashboards");
-  cy.contains('div', DASHBOARD_TITLE)
-    .should('exist').click();
-})
+  cy.visit('dashboards');
+  cy.contains('div', DASHBOARD_TITLE);
```

Comment:
I don't reckon it matters (maybe it was wrong in the original tests), but you are no longer clicking the element.

Reply:
I removed the click since it wasn't needed, this was a mistake from when I originally created this test.